src/main/resources/leonardo-example.conf

```diff
@@ -0,0 +1,113 @@
+# Example conifg file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and secruity certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  applicationName = "leonardo"
+  dataprocDefaultRegion = "us-central1"
+  leoGoogleProject = ""
+  clusterUrlBase = "https://leonardo.dsde-dev.broadinstitute.org/notebooks/"
+  jupyterServerName = "jupyter-server"
+}
+
+# Keys and certificate authorities for cluster
+clusterFiles {
+  configFolderPath = "/etc/"
+  jupyterServerCrt = "jupyter-server.crt"
+  jupyterServerKey = "jupyter-server.key"
+  jupyterRootCaPem = "rootCA.pem"
+  jupyterRootCaKey = "rootCA.key"
+}
+
+# These are the cluster defaults that have already been configured.
+# You should only set them in your config if you want a different value
+#clusterDefaults {
+  #numberOfWorkers = 0
+  #masterMachineType = "n1-standard-4"
+  #masterDiskSize = 500
+  #workerMachineType = "n1-standard-4"
+  #workerDiskSize = 500
+  #numberOfWorkerLocalSSDs = 0
+  #numberOfPreemptibleWorkers = 0
+#}
+
+# Database connection information
+mysql {
+  profile = "slick.jdbc.MySQLProfile$"
+  batchSize = 2000
+  db {
+    driver = "com.mysql.cj.jdbc.Driver"
+    url = "jdbc:mysql://<host>/leonardo?requireSSL=true&useSSL=true&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
+    user = "<user>"
+    password = "<password>"
+    connectionTimeout = 5000
+    numThreads = 50
+  }
+}
+
+proxy {
+  # Should match the jupyter wildcard cert
+  jupyterDomain = ".jupyter.firecloud.org"
+}
+
+# To use the swagger page, supply a service account id and realm
+#swagger {
+  #googleClientId = ""
+  #realm = ""
+#}
+
+akka {
+  loglevel = INFO
+  http {
+    server.idle-timeout = 180 s
+    server.request-timeout=60 s
```

Comment:
Double space `foo = 60s`

Reply:
Also the idle timeouts differ slightly from the ones we use in firecloud-develop. I had bumped them to 1 hour because they were killing our websocket proxy.

(fine to keep request timeout at 60s)

Reply:
the akka block should go in reference.conf if typesafe's silly rules will allow it